docs/components/integration-framework/connectors/out-of-the-box-connectors/googledrive.md

```diff
@@ -0,0 +1,245 @@
+
+1. Set required credentials in the **Authentication** section. See the [relevant appendix entry](#how-can-i-authenticate-my-connector) to find out more.
+2. In the **Select Operation** section, set the field value **Operation Type** as **Create File from template**.
+3. In the **Operation Details** section, set the field **File name** as the desired name of a folder you wish to create. You can use FEEL expressions here.
+4. In the **Operation Details** section, set the field **Template ID** of the desired template.
+5. _(optional)_ In the **Operation Details** section, set the field **Parent folder ID** to the desired parent, inside which a new file will be created. Keep in mind that if not specified, a new folder will be created in the Google Drive root folder of a user who owns the OAuth token.
+6. In the **Operation Details** section, set the field **Template variables** as desired variables that will be applied to the template. The template variables a compatible with the Google Docs Requests API. This property requires FEEL input.
+7. _(optional)_ In the **Operation Details** section, you can set the **Additional properties or metadata** field to Google Drive compatible properties. This property requires FEEL input. Check [the appendix](#what-are-the-limitations-of-the-additional-properties-or-metadata) for known values and limitations.
+
+## Google Drive Connector response
+
+The **Google Drive Connector** exposes Google Drive API response as a local variable called response.
+The following fields are available in the response variable:
+
+- `googleDriveResourceId` - ID of the newly created resource.
+- `googleDriveResourceUrl` - Human-readable URL of the newly created resource.
+
+You can use an output mapping to map the response:
+
+1. Use **Result Variable** to store the response in a process variable. For example, `myResultVariable`.
+2. Use **Result Expression** to map fields from the response into process variables. For example:
+
+```
+= {
+  "myNewReportFolderId": response.googleDriveResourceId,
+  "myNewReportFolderUrl": response.googleDriveResourceUrl
+}
+```
+
+## Appendix & FAQ
+
+### What Google API does the Google Drive Connector use to create a folder?
+
+The **Google Drive Connector** uses Google Drive [`Files:Create`](https://developers.google.com/drive/api/v3/reference/files/create) API endpoint.
+
+### What Google API does the Google Drive Connector use to create a file from template?
+
+The **Google Drive Connector** uses Google Drive [`Files:Copy`](https://developers.google.com/drive/api/v3/reference/files/copy) API endpoint to copy an original template. Afterwards, the **Google Drive Connector** utilizes Google Docs [Merge](https://developers.google.com/docs/api/how-tos/merge) approach via [`Documents:BatchUpdate`](https://developers.google.com/docs/api/reference/rest/v1/documents/batchUpdate) Google Docs API method.
```

Comment:
Pls update first sentence to: "The Google Drive Connector uses the Google Drive...."

Reply:
Addressed in a new revision. Will publish shortly.